libmobilecoin/Cargo.toml

```diff
@@ -14,6 +14,7 @@ aes-gcm = "0.9.4"
 displaydoc = "0.2"
 libc = "0.2"
 protobuf = "2.22.1"
+crc = "1.8.1"
```

Comment:
Why 1.8.1 and not the latest 2.1.0?

Reply:
I was working off an older commit, but you're right it should be 2.1.0